contracts/0.4.24/Lido.sol

```diff
@@ -7,255 +7,326 @@ pragma solidity 0.4.24;
     using StakeLimitUnstructuredStorage for bytes32;
     using StakeLimitUtils for StakeLimitState.Data;
+    using PositiveTokenRebaseLimiter for LimiterState.Data;
 
     /// ACL
-    bytes32 constant public PAUSE_ROLE = keccak256("PAUSE_ROLE");
-    bytes32 constant public RESUME_ROLE = keccak256("RESUME_ROLE");
-    bytes32 constant public STAKING_PAUSE_ROLE = keccak256("STAKING_PAUSE_ROLE");
-    bytes32 constant public STAKING_CONTROL_ROLE = keccak256("STAKING_CONTROL_ROLE");
-    bytes32 constant public MANAGE_FEE = keccak256("MANAGE_FEE");
-    bytes32 constant public MANAGE_WITHDRAWAL_KEY = keccak256("MANAGE_WITHDRAWAL_KEY");
-    bytes32 constant public MANAGE_PROTOCOL_CONTRACTS_ROLE = keccak256("MANAGE_PROTOCOL_CONTRACTS_ROLE");
-    bytes32 constant public BURN_ROLE = keccak256("BURN_ROLE");
-    bytes32 constant public DEPOSIT_ROLE = keccak256("DEPOSIT_ROLE");
-    bytes32 constant public SET_EL_REWARDS_VAULT_ROLE = keccak256("SET_EL_REWARDS_VAULT_ROLE");
-    bytes32 constant public SET_EL_REWARDS_WITHDRAWAL_LIMIT_ROLE = keccak256(
-        "SET_EL_REWARDS_WITHDRAWAL_LIMIT_ROLE"
-    );
-
-    uint256 constant public PUBKEY_LENGTH = 48;
-    uint256 constant public WITHDRAWAL_CREDENTIALS_LENGTH = 32;
-    uint256 constant public SIGNATURE_LENGTH = 96;
-
-    uint256 constant public DEPOSIT_SIZE = 32 ether;
-
-    uint256 internal constant DEPOSIT_AMOUNT_UNIT = 1000000000 wei;
-    uint256 internal constant TOTAL_BASIS_POINTS = 10000;
-
-    /// @dev default value for maximum number of Ethereum 2.0 validators registered in a single depositBufferedEther call
-    uint256 internal constant DEFAULT_MAX_DEPOSITS_PER_CALL = 150;
-
-    bytes32 internal constant FEE_POSITION = keccak256("lido.Lido.fee");
-    bytes32 internal constant TREASURY_FEE_POSITION = keccak256("lido.Lido.treasuryFee");
-    bytes32 internal constant INSURANCE_FEE_POSITION = keccak256("lido.Lido.insuranceFee");
-    bytes32 internal constant NODE_OPERATORS_FEE_POSITION = keccak256("lido.Lido.nodeOperatorsFee");
-
-    bytes32 internal constant DEPOSIT_CONTRACT_POSITION = keccak256("lido.Lido.depositContract");
-    bytes32 internal constant ORACLE_POSITION = keccak256("lido.Lido.oracle");
-    bytes32 internal constant NODE_OPERATORS_REGISTRY_POSITION = keccak256("lido.Lido.nodeOperatorsRegistry");
-    bytes32 internal constant TREASURY_POSITION = keccak256("lido.Lido.treasury");
-    bytes32 internal constant INSURANCE_FUND_POSITION = keccak256("lido.Lido.insuranceFund");
-    bytes32 internal constant EL_REWARDS_VAULT_POSITION = keccak256("lido.Lido.executionLayerRewardsVault");
-
+    bytes32 public constant PAUSE_ROLE = keccak256("PAUSE_ROLE");
+    bytes32 public constant RESUME_ROLE = keccak256("RESUME_ROLE");
+    bytes32 public constant STAKING_PAUSE_ROLE = keccak256("STAKING_PAUSE_ROLE");
+    bytes32 public constant STAKING_CONTROL_ROLE = keccak256("STAKING_CONTROL_ROLE");
+    bytes32 public constant BURN_ROLE = keccak256("BURN_ROLE");
```

Comment:
Consider to delete this - DAO could use Burner contract

Reply:
done